src/python/pants/pantsd/pants_daemon.py

```diff
@@ -75,14 +76,21 @@ class StartupFailure(Exception):
   class RuntimeFailure(Exception):
     """Represents a pantsd failure at runtime, usually from an underlying service failure."""
 
+  class Handle(datatype([('pid', int), ('port', int)])):
+    """A handle to a "probably running" pantsd instance.
+
+    We attempt to verify that the pantsd instance is still running when we create a Handle, but
+    after it has been created it is entirely process that the pantsd instance perishes.
```

Comment:
Probably meant "...entirely possible that" here? I think this is a very useful note to make.